docs/src/content/docs/commands/JSON.MGET.md

```diff
@@ -29,34 +36,36 @@ When the `JSON.MGET` command is executed, DiceDB will:
 3. Extract the value at the specified JSON path from each document.
 4. Return an array of the extracted values.
 
-If a key does not exist or the specified path is not found within a JSON document, `null` will be returned for that key.
+If a key does not exist or the specified path is not found within a JSON document, `nil` will be returned for that key.
 
-## Error Handling
+## Errors
 
 The following errors may be raised by the `JSON.MGET` command:
 
-- `WRONGTYPE Operation against a key holding the wrong kind of value`: This error occurs if one of the specified keys does not hold a JSON document.
-- `ERR syntax error`: This error occurs if the command syntax is incorrect, such as missing parameters or incorrect JSON path format.
+| Error Type                        | Description                                                                                                            |
+|-----------------------------------|------------------------------------------------------------------------------------------------------------------------|
+| `WRONGTYPE`                       | Operation against a key holding the wrong kind of value: This error occurs if one of the specified keys does not hold a JSON document. |
+| `ERR syntax error`                | This error occurs if the command syntax is incorrect, such as missing parameters or incorrect JSON path format.      |
```

Comment:
Please follow this format for errors: https://dicedb.io/commands/jsonnumincrby/#errors